src/data/contributors/AtharvaKhedkar.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "Atharva Khedkar",
+  "skills": ["Python ", "Flutter", "Machine Learning"],
+  "cause": "I code for Passion",
```

Comment:
`I code for` is already the part of string, `passion` would be enough.

Reply:
Made the necessary changes